maps/torch/items/summarydocuments.dm

```diff
@@ -1,393 +1,375 @@
 
-//needs to be rewritten
-///obj/item/document/generic/explo/holocalletiquette
-//	name = "holocall etiquette
-//	desc = "\"Internal use ONLY\" reports, containing a small guide on proper use of holocalls from the perspective of the SEV Charon. This one details five points. First, Calls are a one-way interaction. You inform them of the basic situation, they inform you of what they can do to help. Second, Calls are two-way if you request them to be. Third, If an executive decision is required, ask the Senior on the ground. If he does not render a decision, the SEV Torch shall, but with an emphasis on being decisive. Fourth, State aloud when the call connects. It helps cut down on background chatter. Fifth, The SEV Torch doesn't need to be informed on every specific thing Exploration is doing while on Holocall."
-
-/obj/item/document/generic/explo/holocallloiter
-	name = "holocall etiquette
-	desc = "\"Internal use ONLY\" reports, requesting people that if a holocall lingers, they should make a point to end it.
+/obj/item/documents/generic/explo/cigarettes	
+	name = "cigarettes with helmets"	
+	desc = "Some \"internal use only\" reports detailing the extreme hazards of attempting to smoke a cigarette while wearing a spaceproof helmet, citing many incidents of suffocation and subsequent death as well as pointing out the presence of a \"No Smoking\" sign onboard the Charon and the likely flammability of many exoplanets. It concludes with a recommendation to smoke only in designated breakrooms aboard the SEV Torch and not when wearing a helmet."
 
 /obj/item/documents/generic/explo/documentplease
 	name = "on documentation of sites"
-	desc = "\"Internal Use ONLY\" reports discussing missing information on an exoplanet based on a lack of samples from the exploration team. It discusses the importance of making sure away teams document and report on the exoplanet, particularly taking samples and scans of plant life, xenofauna, atmospheric information, and mineral samples."
+	desc = "Some \"internal use only\" reports discussing missing information on an exoplanet based on a lack of samples from the exploration team. It discusses the importance of making sure away teams document and report on the exoplanet, particularly taking samples and scans of plant life, xenofauna, atmospheric information, and mineral samples."
 
 /obj/item/documents/generic/sci/slimeincident
 	name = "internal corporate communications"
```

Comment:
Corporate associates it too much with the Liaison. 
Suggest changing "corporate" to "department"